strapi/src/components/blocks/opening-hours-item.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Is this enough? We have a specific field for each day now.

Reply:
![image](https://user-images.githubusercontent.com/104828482/210241924-e82688b0-623f-4981-adfe-4f9d057d6dae.png)